### PR TITLE
FLO-1565: location search; not in england service update

### PR DESCRIPTION
**Pull request by teddmason**
is-england model

### CODE REVIEW

File: test/routes/is-england.js [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+      method: 'GET',
+      url: '/is-england/362066/387295'
+    }
+    // Mock the postgres database query
+    server.ext('onPreHandler', (request, reply) => {
+      request.pg = {
+        client: {
+          query: (var1, var2, callback) => {
+            process.nextTick(() => {
+              callback(new Error('database error', null))
```
Comment (davidjamesstone) [inline test/routes/is-england.js]: Should this be (new Error('database error'), null)

File: server/routes/is-england.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    description: 'Returns if Easting and Northing is within England polygon',
+    handler: function (request, reply) {
+      var db = request.pg.client
+      var params = request.params
+      services.isEngland(db, params.x, params.y, function (err, result) {
+        if (err) {
+          return reply(Boom.badImplementation('Database call failed', err))
+        }
+        if (!result || !Array.isArray(result.rows) || result.rows.length !== 1) {
+          return reply(Boom.badRequest('Invalid result'), new Error('Expected an Array'))
```
Comment (davidjamesstone) [inline server/routes/is-england.js]: Should this be: reply(Boom.badRequest('Invalid result', new Error('Expected an Array')))

Comment (teddmason) [inline server/routes/is-england.js]: Nope

File: test/routes/is-england.js (hunk 1 above)
Comment (teddmason) [inline test/routes/is-england.js]: Why wouldn't i send the error back as the first param of the callback?

Comment (teddmason) [inline test/routes/is-england.js]: ah i see, parenthesis in wrong place

**Comment by teddmason** [comment]
@davidjamesstone fixed both of them thanks.  Found a couple more examples as it was a copy and paste issue.

**Pull request merged**